RevenueCatUI/UIKit/PaywallViewController.swift

```diff
@@ -13,10 +13,12 @@
 
 // swiftlint:disable file_length
 
+import SwiftUI
```

Comment:
Any reason we need to move this outside the `#if`?

Reply:
Ahhh yeah, for the `applyIf` that are at the bottom of the file 😅 Its not clear and those should probably get moved somewhere else at some point 🤷‍♂️